ovos_utils/time.py

```diff
@@ -0,0 +1,104 @@
+from datetime import datetime
+from dateutil.tz import gettz, tzlocal
+
+try:
+    from ovos_config.locale import get_default_tz
+    _default_tz = get_default_tz()
+except ImportError:
+    _default_tz = tzlocal()
```

Comment:
we want `get_default_tz` defined here, something like

```
def get_default_tz():
    try:
             from ovos_config.locale import get_default_tz
             return get_default_tz()
    except:
            (TODO _default_tz handling)
```

and in method below always use `get_default_tz` instead of `_default_tz`

this will reflect any config changes and such real time, otherwise _default_tz is stuck to whatever it was set too at load time

Reply:
i wonder if we shouldn't import `get_config_tz` instead

ie 
```python
def get_default_tz():
    try:
        from ovos_config.locale import get_config_tz
        return get_config_tz() or tzlocal()                         # as this maybe is undefined
    except:
        (TODO _default_tz handling)
```
completely eliminating `_default_tz` in the locale module. Or do we need it there?

For the TODO i propose:

```python
_default_tz = None

def get_default_tz():
    ...
    except ImportError:
        return _default_tz or tzlocal()

def set_default_tz(tz=None):
    _default_tz = tz or tzlocal()
```

Reply:
it should use the _default_tz and also wrap the set_default_tz method for now, that sort of change should be discussed in ovos-config first and only if updated there then reflected here

Reply:
actually changed my mind, in general im thinking the whole locale module should go away from ovos-config at some point

the concept of lf active lang/tz makes no sense anymore now that we are multilingual and are moving away from LF completely

go ahead with change so we can start deprecating that by importing from utils

do not port the get/set  default_tz methods, only get_config_tz